server/controllers/UserController.js

```diff
@@ -62,3 +62,9 @@ exports.delete = async (req, res) => {
   await User.destroy({ where: { id } });
   res.status(204).send();
 };
+
+exports.getHeaders = async (req, res) => {
```

Comment:
Mark this as development only and ideally comment it out.

Reply:
we do not need that anymore